From 39d7fb64a0fb9877c1e24023abaf992ba66c7b08 Mon Sep 17 00:00:00 2001
From: Hiroshi Hatake <hatake@clear-code.com>
Date: Mon, 27 Aug 2018 15:22:05 +0900
Subject: [PATCH] Force set configuration directories

---
 cmake/FcitxConfig.cmake | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/cmake/FcitxConfig.cmake b/cmake/FcitxConfig.cmake
index df0f3416..40976faa 100644
--- a/cmake/FcitxConfig.cmake
+++ b/cmake/FcitxConfig.cmake
@@ -89,4 +89,9 @@ ENDIF(FCITX4_FCITXCONFIG_EXECUTABLE)
 INCLUDE(FindPackageHandleStandardArgs)
 FIND_PACKAGE_HANDLE_STANDARD_ARGS(Fcitx DEFAULT_MSG FCITX4_VERSION)
 
+set(FCITX4_ADDON_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/share/fcitx/addon")
+set(FCITX4_INPUTMETHOD_CONFIG_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/share/fcitx/inputmethod")
+set(FCITX4_CONFIGDESC_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/share/fcitx/inputmethod")
+set(FCITX4_ADDON_CONFIG_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/share/fcitx/addon")
+
 INCLUDE(${CMAKE_SYSROOT}${FCITX4_PREFIX}/share/cmake/fcitx/FcitxMacro.cmake)
-- 
2.11.0

