From ad16f5b050fe5c283d3c0b1139448e882b3340b9 Mon Sep 17 00:00:00 2001
From: Takuro Ashie <ashie@clear-code.com>
Date: Tue, 21 Sep 2021 16:46:09 +0900
Subject: [PATCH] Avoid to add CMAKE_SYSROOT to ISOCODES_ISO*_XML

Signed-off-by: Takuro Ashie <ashie@clear-code.com>
---
 cmake/FindIsoCodes.cmake | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/cmake/FindIsoCodes.cmake b/cmake/FindIsoCodes.cmake
index 384c82bc..9a246f1f 100644
--- a/cmake/FindIsoCodes.cmake
+++ b/cmake/FindIsoCodes.cmake
@@ -21,10 +21,16 @@ pkg_check_modules(PC_ISOCODES iso-codes)
 find_file(ISOCODES_ISO639_XML iso_639.xml
           HINTS "${PC_ISOCODES_PREFIX}/share/xml/iso-codes/"
           )
+if (ISOCODES_ISO639_XML)
+  set(ISOCODES_ISO639_XML "${PC_ISOCODES_PREFIX}/share/iso-codes/json/iso_639.xml")
+endif()
 
 find_file(ISOCODES_ISO3166_XML iso_3166.xml
           HINTS "${PC_ISOCODES_PREFIX}/share/xml/iso-codes/"
           )
+if (ISOCODES_ISO3166_XML)
+  set(ISOCODES_ISO3166_XML "${PC_ISOCODES_PREFIX}/share/iso-codes/json/iso_3166.xml")
+endif()
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(IsoCodes  DEFAULT_MSG  ISOCODES_ISO639_XML ISOCODES_ISO3166_XML)
-- 
2.30.2

