From 4f081a9b844410f5e2d191dfe4421693545af297 Mon Sep 17 00:00:00 2001
From: Takuro Ashie <ashie@clear-code.com>
Date: Tue, 27 Jul 2021 11:48:44 +0900
Subject: [PATCH 1/2] Add CMAKE_SYSROOT to find additional cmake file

Signed-off-by: Takuro Ashie <ashie@clear-code.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 12b9fca..a2d829a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -22,7 +22,7 @@ else()
     set(ANTHY_TARGET PkgConfig::Anthy)
 endif()
 
-include("${FCITX_INSTALL_CMAKECONFIG_DIR}/Fcitx5Utils/Fcitx5CompilerSettings.cmake")
+include("${CMAKE_SYSROOT}${FCITX_INSTALL_CMAKECONFIG_DIR}/Fcitx5Utils/Fcitx5CompilerSettings.cmake")
 add_definitions(-DFCITX_GETTEXT_DOMAIN=\"fcitx5-anthy\" -D_GNU_SOURCE)
 fcitx5_add_i18n_definition()
 
-- 
2.30.2

