--- a/src/modules/spell/dict/CMakeLists.txt       2019-08-26 01:11:04.000000000 +0000
+++ b/src/modules/spell/dict/CMakeLists.txt       2021-06-06 12:05:19.224349815 +0000
@@ -3,7 +3,13 @@
   )
 
 add_executable(comp-spell-dict ${DICT_COMP_SRC})
-add_executable(Fcitx5::comp-spell-dict ALIAS comp-spell-dict)
+
+if("${FCITX_TOOL_BINARY_DICT}" STREQUAL "")
+       add_executable(Fcitx5::comp-spell-dict ALIAS comp-spell-dict)
+else()
+       add_executable(Fcitx5::comp-spell-dict ALIAS "${FCITX_TOOL_BINARY_DIR}/libexec/comp-spell-dict")
+endif()
+
 target_link_libraries(comp-spell-dict Fcitx5::Utils)
 install(TARGETS comp-spell-dict
   DESTINATION "${FCITX_INSTALL_ADDONDIR}/libexec")
