From 7ce9f8ef092cfd70754b054395ef19f82a39f1eb Mon Sep 17 00:00:00 2001
From: Takuro Ashie <ashie@clear-code.com>
Date: Tue, 20 Jul 2021 16:12:10 +0900
Subject: [PATCH 5/8] Avoid to add CMAKE_SYSROOT to ISOCODES_ISO*_JSON

Signed-off-by: Takuro Ashie <ashie@clear-code.com>
---
 cmake/FindIsoCodes.cmake | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/cmake/FindIsoCodes.cmake b/cmake/FindIsoCodes.cmake
index d2d1318..a9e1377 100644
--- a/cmake/FindIsoCodes.cmake
+++ b/cmake/FindIsoCodes.cmake
@@ -21,10 +21,16 @@ pkg_check_modules(PC_ISOCODES iso-codes)
 find_file(ISOCODES_ISO639_JSON iso_639-3.json
           HINTS "${PC_ISOCODES_PREFIX}/share/iso-codes/json/"
           )
+if (ISOCODES_ISO639_JSON)
+  set(ISOCODES_ISO639_JSON "${PC_ISOCODES_PREFIX}/share/iso-codes/json/iso_639-3.json")
+endif()
 
 find_file(ISOCODES_ISO3166_JSON iso_3166-1.json
           HINTS "${PC_ISOCODES_PREFIX}/share/iso-codes/json/"
           )
+if (ISOCODES_ISO3166_JSON)
+  set(ISOCODES_ISO3166_JSON "${PC_ISOCODES_PREFIX}/share/iso-codes/json/iso_3166-1.json")
+endif()
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(IsoCodes  DEFAULT_MSG  ISOCODES_ISO639_JSON ISOCODES_ISO3166_JSON)
-- 
2.30.2

