From 3586f950cdd4ac153ab0c93f64ece78d54c03991 Mon Sep 17 00:00:00 2001
From: Takuro Ashie <ashie@clear-code.com>
Date: Tue, 27 Jul 2021 22:56:42 +0900
Subject: [PATCH] Enable to build against older GLib (< 2.60)

Signed-off-by: Takuro Ashie <ashie@clear-code.com>
---
 gtk3/fcitximcontext.cpp | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/gtk3/fcitximcontext.cpp b/gtk3/fcitximcontext.cpp
index 3e62f1f..98b4841 100644
--- a/gtk3/fcitximcontext.cpp
+++ b/gtk3/fcitximcontext.cpp
@@ -465,6 +465,10 @@ static void fcitx_im_context_init(FcitxIMContext *context, gpointer) {
     g_queue_init(&context->gdk_events);
 }
 
+static void free_gdk_event(gpointer gdkevent, gpointer user_data) {
+  gdk_event_free((GdkEvent*)gdkevent);
+}
+
 static void fcitx_im_context_finalize(GObject *obj) {
     FcitxIMContext *context = FCITX_IM_CONTEXT(obj);
     delete context->candidate_window;
@@ -487,7 +491,8 @@ static void fcitx_im_context_finalize(GObject *obj) {
     g_clear_pointer(&context->preedit_string, g_free);
     g_clear_pointer(&context->surrounding_text, g_free);
     g_clear_pointer(&context->attrlist, pango_attr_list_unref);
-    g_queue_clear_full(&context->gdk_events, (GDestroyNotify)gdk_event_free);
+    g_queue_foreach(&context->gdk_events, free_gdk_event, NULL);
+    g_queue_clear(&context->gdk_events);
 
     G_OBJECT_CLASS(parent_class)->finalize(obj);
 }
-- 
2.30.2

